doc/getting_started/getting_started.rst

```diff
@@ -116,19 +116,27 @@ The ELF binaries generated by the build system are named by default
 configuration The build system generates different names for different use cases
 depending on the hardware and boards used.
 
-.. _third_party_x_compilers:
 
-Using Custom and 3rd Party Cross Compilers
-==========================================
+.. _sdkless_builds:
+
+Building without the Zephyr SDK
+===============================
 
 The Zephyr SDK is provided for convenience and ease of use. It provides
 cross-compilers for all ports supported by the Zephyr OS and does not require
 any extra flags when building applications or running tests.
 
-If you have a custom cross-compiler or if you wish to use a vendor provided SDK,
-follow the steps below to build with any custom or 3rd party cross-compilers:
+In addition to cross-compilers the Zephyr SDK also provides prebuilt
```

Comment:
add a comma:

    cross-compilers, the

Reply:
Done